components/browser/session/src/main/java/mozilla/components/browser/session/SessionManager.kt

```diff
@@ -207,6 +208,22 @@ class SessionManager(
      */
     fun findSessionById(id: String): Session? = values.find { session -> session.id == id }
 
+    /**
+     * Tells to [SessionManager] that OS is in low memory conditions and needs
```

Comment:
maybe `Informs this [SessionManager] that the OS is in low...`

Reply:
Your right, "Informs" sounds nicer!